weaviate/collections/classes/config_methods.py

```diff
@@ -181,7 +181,9 @@ def _collection_config_from_json(schema: Dict[str, Any]) -> _CollectionConfig:
             key=schema["shardingConfig"]["key"],
             strategy=schema["shardingConfig"]["strategy"],
             function=schema["shardingConfig"]["function"],
-        ),
+        )
+        if not schema["multiTenancyConfig"]["enabled"]
+        else None,
```

Comment:
what is this change for? You can have multiple shards without MT

Reply:
You can't have sharding and multi-tenancy though, which is what this is doing. I will reverse the logic to make it clearer